client/src/test/java/org/apache/rocketmq/client/impl/consumer/DefaultLitePullConsumerImplTest.java

```diff
@@ -0,0 +1,85 @@
+package org.apache.rocketmq.client.impl.consumer;
+
+import org.apache.rocketmq.client.consumer.DefaultLitePullConsumer;
+import org.apache.rocketmq.common.message.MessageQueue;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * @author Willhow
+ * @since 2024/5/3
+ */
```

Comment:
Author information needs to be removed

Reply:
already removed.Idea generated it automatically, so I didn’t notice it.

Reply:
> 作者信息需要删除

already removed. Also revised code style and added license comments